Helper/ApiData.php

```diff
@@ -202,7 +201,7 @@ public function getResponse(
                 $response["data"] = $e->getMessage();
                 $response["status"] = $e->getCode();
             }
-            throw new CheckoutException(__('Connection error to OP Payment Service API'));
```

Comment:
This fix in SQMAGOPC-135 caused the problem with online refunds. Need to reimplement some handling for incorrect Api keys.

Reply:
No sure, if the status should not be chcked in Controller/Redirect/Index.php (line 81, $responseData = $this->getResponseData($order);) Now, there is try {} catch {} statement, but if exception is not thrown, I am not sure if it get handled correctly

Reply:
The problem with not thrown exception was handled.